sdk/ai/Azure.AI.Inference/CHANGELOG.md

```diff
@@ -1,16 +1,14 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2025-03-18)
 
 ### Features Added
 
 - Added extension methods to get `ChatCompletionsClient` and `EmbeddingsClient` using [AIProjectClient](https://learn.microsoft.com/dotnet/api/azure.ai.projects.aiprojectclient?view=azure-dotnet-preview).
 
-### Breaking Changes
-
 ### Bugs Fixed
 
-### Other Changes
+- Fixed an issue with the audio samples not properly showcasing the intended workflow.
```

Comment:
What "showcasing the intended workflow" means, ie. what kind of issue?

Reply:
Discussed offline